### ISSUE TITLE
[Bert/Tensorflow] hang, keep printing loss and step during training

### ISSUE THREAD

**Issue by wenhuach** [NVIDIA/DeepLearningExamples#425]
Related to **Model/Framework(s)** 
Bert Tensorflow1.14

**Describe the bug**
keep printing loss and step during training

**To Reproduce**
pretrain bert on uncased_L-12_H-768_A-12 with adam

**Expected behavior**
the training will go on 

**Environment**
Please provide at least:
* Container version (e.g. tensorflow1.14):
* GPUs in the system: (e.g. 8x Tesla V100-SXM2-16GB):
* CUDA driver version (e.g. 418.39):
![bug](https://user-images.githubusercontent.com/8307298/77131078-52338180-6a95-11ea-9392-c44387c040e7.jpg)

**Comment by swethmandava** [comment]
Could you attach the full log? Do you mean the training is stuck on step = 83311 in this case?

**Comment by wenhuach** [comment]
Sorry for the delay.  After keep running it, I finally got a truncated record exception
![image](https://user-images.githubusercontent.com/8307298/78033098-857ce700-7398-11ea-9b5c-6b94b300921e.png). 
I have regenerated the data following the instructions of google-bert and yours, since I don't use the docker environment. However, it seems that I still have the same issue. 
By the way, I also met NAN issue during the training.

**Comment by kanhuimin** [comment]
Has your datalosserror problem been solved?

**Comment by BIGKUANG** [comment]
I meet the same problem. 
gpu: v100 official docker env   3card gpu parallel batch_size=32  fp16 
before loss is gradual decline, but on 56340 step the loss jump to about 8.0, after that pretrain is keep,but loss stay close to 8.

<img width="979" alt="91fbc92a49dde7099ed521d3f640ea2" src="https://user-images.githubusercontent.com/23147851/89527977-d6a45800-d81c-11ea-910a-d708ea025a01.png">


I try to use float32 , but is also reproduction.

**Comment by swethmandava** [comment]
#529 also sees a loss jump that could be related. Were you able to root cause / fix the issue? @BIGKUANG
